examples/composed-examples/listInitialAccounts.ts

```diff
@@ -0,0 +1,102 @@
+import { createConcordiumClient, unwrap } from '@concordium/node-sdk';
+import { credentials } from '@grpc/grpc-js';
+import { parseEndpoint } from '../shared/util';
+
+import meow from 'meow';
+
+const cli = meow(
+    `
+  Usage
+    $ yarn run-example <path-to-this-file> [options]
+
+  Required
+    --from, -f  From some time, defaults to genesis time
+    --to,   -t  To some time, defaults to the timestamp for the last finalized block
```

Comment:
It is not clear to me what I have to give as input to avoid errors here. :thinking: 

```
Trace
    at unwrap (/home/hjort/Projects/concordium-node-sdk-js/packages/common/lib/util.js:151:17)
    at file:///home/hjort/Projects/concordium-node-sdk-js/examples/composed-examples/listInitialAccounts.ts:58:24
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
Error: Undefined value found.
    at unwrap (/home/hjort/Projects/concordium-node-sdk-js/packages/common/lib/util.js:152:15)
    at file:///home/hjort/Projects/concordium-node-sdk-js/examples/composed-examples/listInitialAccounts.ts:58:24
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
```

```
Processing blocks...
RangeError: Division by zero
    at file:///home/hjort/Projects/concordium-node-sdk-js/examples/composed-examples/listInitialAccounts.ts:77:49
```